prisma/migrations/20240803171237_setup_schema/migration.sql

```diff
@@ -0,0 +1,270 @@
+-- CreateEnum
+CREATE TYPE "TransactionType" AS ENUM ('DEPOSIT', 'PURCHASE');
+
+-- CreateEnum
```

Comment:
Avoid comments that don't add additional information for reader, comments like `-- CreateTable` is unnecessary since the query clearly indicates `CREATE TABLE`. Same goes for `-- CreateEnum` ya.